frontend/src/App.res

```diff
@@ -108,10 +108,15 @@ module Benchmark = {
   @react.component
   let make = React.memo((~repoId, ~pullNumber, ~data: GetBenchmarks.t, ~lastCommit) => {
     let benchmarkDataByTestName = React.useMemo2(() => {
-      data.benchmarks->makeBenchmarkData->AdjustMetricUnit.adjust
+      data.benchmarks->makeBenchmarkData->AdjustMetricUnit.adjust->AppHelpers.fillMissingValues
     }, (data.benchmarks, makeBenchmarkData))
+
     let comparisonBenchmarkDataByTestName = React.useMemo2(
-      () => data.comparisonBenchmarks->Belt.Array.reverse->makeBenchmarkData,
+      () =>
+        data.comparisonBenchmarks
+        ->Belt.Array.reverse
+        ->makeBenchmarkData
+        ->AppHelpers.fillMissingValues,
```

Comment:
Do we need to call `AdjustMetricUnit.adjust` like on line 111?

Reply:
I noticed this too, while making the change. I think the `adjust` function would need to be called on both the benchmark and the comparison values together, otherwise potentially both the values could be adjusted to different units. I'll make a separate issue for this.

Reply:
Opened #322